webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricsSharedList.js

```diff
@@ -45,7 +80,6 @@ export class SakaiRubricsSharedList extends SakaiRubricsList {
   }
 
   getSharedRubrics() {
-
```

Comment:
Please leave these blank lines in. I know it seems a bit pedantic, but I think it makes js easier to read to leave a blank like on a multiline function definition - I do that everywhere in the webcomponents stuff.